# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/tibit/ggrab/00-makefile.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -ur ggrab-0.21/Makefile patched-ggrab-0.21/Makefile
--- ggrab-0.21/Makefile	2003-01-06 13:09:20.000000000 +0100
+++ patched-ggrab-0.21/Makefile	2003-02-21 14:47:59.000000000 +0100
@@ -13,12 +13,12 @@
 #
 #
 
-CPPFLAGS=-DREENTRANT -D_LARGEFILE64_SOURCE  -Wall -Wshadow -O3
+CPPFLAGS=-DREENTRANT -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 -Wall -Wshadow -O3
 
-LDFLAGS= -s -static
+LDFLAGS= -s
 
 INSTALL=/usr/bin/install 
-DESTDIR=/usr/local
+DESTDIR=/usr
 
 TARGETS=ggrab sserver
 
